Makefile

```
@@ -1,55 +1,23 @@
-VERSION := $(shell grep '^version:' pubspec.yaml | sed -e 's/version: //')
-DIST_ANDROID := ./dist/android/$(VERSION)
-DIST_IOS := ./dist/ios/$(VERSION)
-# DELETE .DS_Store file from all release
+.PHONY: android, ios
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)